File: 6. Python/EX43.py
# Inicializa a soma acumulada
acuSom = 0

# Solicita a quantidade de números a serem digitados
n = int(input("# Quantos numeros voce vai digitar: "))

# Cria um vetor de tamanho n preenchido com zeros
vet = [0.0 for i in range(n)]

# Loop para receber os números e calcular a soma
for i in range(n):

    # Lê um número e o armazena no vetor
    vet[i] = float(input("# Digite um numero: "))

    # Acumula a soma dos números
    acuSom += vet[i]

# Exibe os valores digitados
print("\n# VALORES = ", end="")
for i in range(n):

    # Formata a saída para duas casas decimais
    print(f"{vet[i]:.2f}", end=" ")

# Exibe a soma dos valores
print(f"\n# Soma = {acuSom:.2f}")

# Exibe a média dos valores
print(f"# Media = {(acuSom/n):.2f}")

File: 4. Java/EX43.java
package Apostila;

import java.util.Locale;
import java.util.Scanner;

public class EX43
{
    public static void main(String[] args)
    {
        // Quantidade e soma
        int n;
        double acuSom = 0;
        
        // Define Locale
        Locale.setDefault(Locale.US);
        
        // Entrada de dados
        try (Scanner ent = new Scanner(System.in))
        {
            // Solicita quantidade
            System.out.print("# Quantos numeros voce vai digitar: ");
            n = ent.nextInt();
            
            // Vetor de números
            double vet[] = new double[n];
            
            // Recebe números e acumula soma
            for (int i = 0; i < n; i++)
            {
                System.out.print("# Digite um numero: ");
                vet[i] = ent.nextDouble();
                acuSom += vet[i];
            }
            
            // Exibe valores
            System.out.print("\n# VALORES =");
            for (int i = 0; i < n; i++)
            {
                System.out.print(" " + vet[i]);
            }
            
            // Exibe soma e média
            System.out.printf("\n# SOMA = %.2f\n", acuSom);
            System.out.printf("# MEDIA = %.2f\n", (acuSom/n));
        }
    }
}
